public/app/features/scenes/dashboard/DashboardsLoader.ts

```diff
@@ -264,6 +264,10 @@ function createVizPanelFromPanelModel(panel: PanelModel) {
     pluginVersion: panel.pluginVersion,
     $data: new SceneQueryRunner({
       queries: panel.targets,
+      timeFrom: panel.timeFrom,
+      timeShift: panel.timeShift,
+      hideTimeOverride: panel.hideTimeOverride,
```

Comment:
Thinking out loud - maybe we should move the `hideTimeOverride` to VizPanel config straight away? Regardless of how the time shift will be configured in the future (ref @torkelo [comment](https://github.com/grafana/grafana/pull/60213#pullrequestreview-1233152993)), I think this is pure panel config option.

Reply:
I agree, it is a display option more than a query one. I just replicated what we had, but I think as VizPanel prop makes more sense. WDYT @torkelo?

Reply:
Yes, was thinking the same